README.md

```diff
@@ -62,6 +62,16 @@ rules_oci supports two different registry implementation for the temporary stora
 
 ## Public API Docs
 
+### Build Base images
+
+- Alpine: we recommend <https://github.com/chainguard-dev/rules_apko> to
+  install [apk](https://wiki.alpinelinux.org/wiki/Package_management) packages
+  using Chainguard's [apko](https://apko.dev).
+- Debian: The `apt-get` utility installs `.deb` files, which are already archives
+  that may be used directly as image layers. See `/examples/deb` in this repository.
+  Note that you must enumerate every `.deb` file to install, including transitive dependencies.
```

Comment:
nit: also mention `/var/lib/dpkg/status.d` entries must be created manually for each deb. https://github.com/GoogleContainerTools/distroless/blob/0aaf3a9a08c66b7f3dda323be174251a98e11ced/private/pkg/dpkg_status.go#L17